tests/console/sshfs.pm

```diff
@@ -27,8 +27,8 @@ sub run() {
     send_key "ret";
     assert_screen 'sshfs-accepted';
     script_run('cd mnt/tmp');
-    assert_script_run("zypper -n in xdelta");
-    assert_script_run("rpm -e xdelta");
+    assert_script_run("zypper -n in xdelta3");
+    assert_script_run("rpm -e xdelta3");
```

Comment:
`select * from job_modules where (name='console-sshfs') limit 10;` did not return anything from the osd database and also sshfs is only mentioned in `products/opensuse/main.pm` so we are safe :-)